package.json

```
@@ -13,14 +13,15 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/CMSgov/macpro-quickstart-serverless.git"
+    "url": "https://github.com/CMSgov/mdct-mcr"
```

nit: The repository should be the value that you would pass to git clone. For `https` clones that would be `https://github.com/CMSgov/mdct-mcr.git`
```suggestion
    "url": "https://github.com/CMSgov/mdct-mcr.git"
```

---

Good catch. That one's my bad for just copy/pasting and forgetting to add the `.git`.